polaris/tests/test_layers.py

```diff
@@ -0,0 +1,34 @@
+import pytest
```

Comment:
add this module to the requirements.txt

Reply:
I thought about it. Since our library does not need this dependency, I only install it for Travis CI.